include/cvc5/cvc5.h

```diff
@@ -4293,13 +4349,27 @@ class CVC5_EXPORT Solver
    * @warning This method is experimental and may change in future versions.
    *
    * @param c The component of the proof to return
-   * @return A string representing the proof. This takes into account
-   * :ref:`proof-format-mode <lbl-option-proof-format-mode>` when `c` is
-   * `PROOF_COMPONENT_FULL`.
+   * @return A vector of proof nodes.
```

Comment:
A `Proof` is not a `node`, is it? We don't really have a notion of `node` on the API level.